V6_MIGRATION.md

```diff
@@ -151,3 +154,13 @@ cardClient.tokenizeCard(card) { (tokenizedCard, error) in
     // Communicate the tokenizedCard.nonce to your server, or handle error
 }
 ```
+
+## SEPA Direct Debit
+`BTSEPADirectDebitClient.tokenize(request: BTSEPADirectDebitRequest, context: ASWebAuthenticationPresentationContextProviding)` has been renamed to `BTSEPADirectDebitClient.tokenize(request: BTSEPADirectDebitRequest)`. Additionally, conformance to the `ASWebAuthenticationPresentationContextProviding` protocol is no longer needed.
```

Comment:
Another take it or leave it comment - instead of "method was renamed to" this makes more sense to me as "removed parameter from".

Reply:
done!